flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/FlinkAggregateExpandDistinctAggregatesRule.java

```diff
@@ -470,7 +470,7 @@ private void rewriteUsingGroupingSets(RelOptRuleCall call, Aggregate aggregate)
                                 aggCall.left.filterArg,
                                 aggregate.getGroupCount(),
                                 fullGroupSet.cardinality());
-                distinctAggCalls.add(newAggCall.rename(aggCall.right));
+                distinctAggCalls.add(newAggCall.withName(aggCall.right));
```

Comment:
deprecated at https://github.com/apache/calcite/commit/513f4d2510a47e8d0b55f35dac56785c16fa7d5f
in favor of `withXXX` methods